_episodes/01-intro.md

```diff
@@ -72,6 +72,11 @@ In the examples for this lesson, we'll show the prompt as `$ `.
 Most importantly:
 when typing commands, either from these lessons or from other sources,
 *do not type the prompt*, only the commands that follow it.
+Also note that after you type a command, you have to press the *Enter* key yo execute it.
```

Comment:
Can you please convert the Enter code to use the `<kbd>` syntax.

Reply:
Done. Also corrected s/yo/to/.